docs/documentation/user-documentation/faq.md

```diff
@@ -108,10 +108,29 @@ I have a negative control
 
 _There is no specific taxonomy terms for positive and negative controls_
 
-## How do fill in negative controls?
+## How do I fill in negative controls?
 
 _TODO_
 
+## How to fill in missing data
+
+Follow the [INSDC Missing Value Reporting](https://www.insdc.org/technical-specifications/missing-value-reporting/) specifications.
+- Missing data is (meta)data that **can not** be reported. This can be due to various constraints, such as:
+  - the data is truly missing, meaning it is not known and can not be known;
+  - it can not be shared for privacy reasons;
+  - it can not be shared due to data agreement restrictions;
+  - it is not applicable to that particular field (e.g. it is a negative control and the field does not apply)
+- Missing data should only be reported for **mandatory** fields, not for **recommended** or **optional** fields. For the latter two, simply leave the field blank if the (meta)data are missing.
```

Comment:
I think the mandatory fields sentence should go at the beginning of this section in a preamble paragraph and say more along the lines of:

Fields in MIxS that are mandatory always require _something_ filled into the given metadata entry. If you do not have this information, you must encode this using the specific 'missing information' categories as below.

While optional fields in MIxS can be left blank, if you have a specific reason the information will never be able to be reported (see examples below), then it is good to use these missing data categories there also.